meta/meta-node/src/command.rs

```diff
@@ -112,7 +112,12 @@ pub fn run() -> sc_cli::Result<()> {
 					backend,
 					..
 				} = service::new_partial(&config)?;
-				Ok((cmd.run(client, backend), task_manager))
+				let aux_revert = Box::new(|_client, _, _blocks| {
```

Comment:
not sure if we need an auxiliary handler for the revert as the finality is also handled by ManualSeal itself. might need some checking and tests to confirm. for now, we can keep it this way as it does nothing extra.